java/org/prism/ParsingOptions.java

```diff
@@ -12,9 +12,10 @@ public abstract class ParsingOptions {
      * See pm_options_version_t in include/prism/options.h.
      */
     public enum SyntaxVersion {
-        LATEST(0),
         V3_3(1),
-        V3_4(2);
+        V3_4(2),
+        V3_5(3),
+        LATEST(V3_5.value);
```

Comment:
Could be:
```suggestion
        LATEST(0);
```
So that there is no risk to forget updating it.
And then 0 would be translated when "parsing" the options (already done in `pm_parser_init` but not sure if that gets called when using the serialization).
Same for `javascript/src/parsePrism.js` below